sources/parse-datetime/README.md

```diff
@@ -9,7 +9,8 @@ This library parses a `DateTime<Utc>` from a string.
 The string can be:
 
 * an `RFC3339` formatted date / time
-* a string with the form `"in <unsigned integer> <unit(s)>"` where
+* a string with the form `"[in] <unsigned integer> <unit(s)>"` where
+   * 'in' is optional
```

Comment:
```suggestion
* a string with the form `"[in] <unsigned integer> <unit(s)>"` where 'in' is optional
```
Don't think that was meant to be a separate bullet?

Reply:
Originally, yes. But this reads better; will fix.